.github/workflows/integration-tests.yml

```diff
@@ -289,6 +289,7 @@ jobs:
 
       - name: Run python tests on HIP
         run: |
+          pytest --capture=tee-sys -rfs -vvv python/tutorial/06-fused-attention.py
```

Comment:
Code and Triton IR of `06-fused-attention.py` and `test_flash_attention.py` is significantly different, in my opinion it make sense to tests both.